lang/src/test/kotlin/org/partiql/lang/compiler/memorydb/operators/GetByKeyProjectRelationalOperatorFactory.kt

```diff
@@ -57,11 +57,17 @@ class GetByKeyProjectRelationalOperatorFactory : ProjectRelationalOperatorFactor
         // Parse the tableId so we don't have to at evaluation-time
         val tableId = UUID.fromString(impl.staticArgs.single().textValue)
 
+        var exhausted = false
```

Comment:
Should the changes in the file be added in this PR?

Reply:
It's just a test, and it would have warned us about the early sequence creation. I'm ok with keeping these changes